<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FG_ConfigDialog</class>
 <widget class="QDialog" name="FG_ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Framegrabber Acquisition</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="browseBut">
   <property name="geometry">
    <rect>
     <x>304</x>
     <y>30</y>
     <width>88</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="outputFileLE">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>281</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="com">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>COM1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM4</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="baud">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>230400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>57600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>38400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>28800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2400</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="bits">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="parity">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>N</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>E</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>O</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>COM Port Setup:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="stop">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data save file:</string>
   </property>
  </widget>
  <widget class="QLabel" name="sapdevLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sapera Device:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sapdevCB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Xtium CL PX-4 Mono</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Configuration:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="janeliaRadio">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Janelia 2304 channels @ 20kHz</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">configBG</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="calinRadio">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Calin's Test - 2048 channels @ 38.4kHz</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">configBG</string>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="disableChanMapChk">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;For Jim Chen testing only. &lt;/p&gt;&lt;p&gt;If checked, disable the &amp;quot;channel mapping code&amp;quot; and simply show electrodes as they come in from the intan chips in the framegrabber frames.&lt;/p&gt;&lt;p&gt;Not that if unchecked, you can always &amp;quot;sort by intan&amp;quot; or &amp;quot;sort by electrode&amp;quot; from the options menu. Sorting by intan should show the channels as they come in from the framegrabber (assuming the channel mapping code is bug-free).&lt;/p&gt;&lt;p&gt;Default is unchecked, which means channel mapping is enabled.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;For Jim Chen testing only. &lt;/p&gt;&lt;p&gt;If checked, disable the &amp;quot;channel mapping code&amp;quot; and simply show electrodes as they come in from the intan chips in the framegrabber frames.&lt;/p&gt;&lt;p&gt;Not that if unchecked, you can always &amp;quot;sort by intan&amp;quot; or &amp;quot;sort by electrode&amp;quot; from the options menu. Sorting by intan should show the channels as they come in from the framegrabber (assuming the channel mapping code is bug-free).&lt;/p&gt;&lt;p&gt;Default is unchecked, which means channel mapping is enabled.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Disable ChanMap</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FG_ConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FG_ConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="configBG"/>
 </buttongroups>
</ui>
